packages/desktop-client/src/components/modals/SelectLinkedAccountsModal.tsx

```diff
@@ -101,14 +132,17 @@ export function SelectLinkedAccountsModal({
     account => !Object.values(chosenAccounts).includes(account.id),
   );
 
-  function onSetLinkedAccount(externalAccount, localAccountId) {
-    setChosenAccounts(accounts => {
-      const updatedAccounts = { ...accounts };
+  function onSetLinkedAccount(
+    externalAccount: NormalizedAccount,
+    localAccountId: string | undefined,
```

Comment:
Could this be optional instead?

```suggestion
    localAccountId?: string,
```